pkgs/top-level/ocaml-packages.nix

```diff
@@ -965,6 +969,9 @@ let
     then callPackage ../development/ocaml-modules/expat { }
     else callPackage ../development/ocaml-modules/expat/0.9.nix { };
 
+    # For backwards compatibilty
+    ocaml_extlib = extlib;
```

Comment:
When will this alias be removed? I suggest to remove it now.

Reply:
Done.